### DISCUSSION TITLE
Does nerdctl use semver?

### DISCUSSION THREAD

**Discussion by stockholmux** [containerd/nerdctl#2489] [Q&A] [answered]
I debated if this should be an issue but I don't see any explicit indication of how nerdctl manages versions. 

It _looks_ like it follows SemVer, but I'd love confirmation (and, ideally, something in the readme or faq).

**Comment by AkihiroSuda** [comment, marked as answer]
Yes

**Comment by stockholmux** [comment]
That's great news! Should it be in the README or FAQ? I'm glad to do a PR.
